deploy/operator-install.yaml

```
@@ -934,21 +934,18 @@ spec:
                         properties:
                           clusterName:
                             type: string
-                          host:
-                            type: string
                           name:
                             type: string
                           namespace:
                             type: string
-                          port:
-                            format: int32
-                            type: integer
                           secretName:
                             type: string
                           url:
+                            pattern: ^(kubernetes|openshift|xsite):\/\/(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])(:[0-9]+)?$
```

```suggestion
                            pattern: ^(minikube|openshift|xsite):\/\/(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])(:[0-9]+)?$
```
`minikube` scheme is documented for kubernetes.
Do we really need a so strict checking for url?

---

No other values are possible. This will reduce number of customer mistakes.
Also proposed changes from `minikube` to `kuberntes` with issue #703 and PR #764

---

I would make this change backward compatible then, to avoid upgrade problem, can we accept both scheme for static conf?

---

Yeah, good idea to keep for backward compatibility. Will update both PR's
Now operator try to access remote site on every config change, but we would like to reflect ot only in case of locations URL or secret changes in the future

---

Also upgrade fails now, it's time to add some breaking changes I think :+1: